docs/guides/analytics/index.md

```diff
@@ -0,0 +1,815 @@
+---
+title: Analytics API
```

Comment:
Why has the "Analytics API" doc been placed into the Developer guides section? I don't think it's consistent. I was expecting it in the API guides (into the "Subsystems section) because it's linked in the /docs/apis page...

Reply:
I originally placed it in subsystems, and then wasn't sure because it seemed more like a guide than a doc.

I have moved it back.